src/Tags/Url.php

```diff
@@ -87,7 +87,7 @@ public function setChangeFrequency(string $changeFrequency)
      */
     public function setPriority(float $priority)
     {
-        $this->priority = $priority;
+        $this->priority = max(0, min(1, $priority));
```

Comment:
I think it would be good to add a test for this newly added logic.

Reply:
done https://github.com/spatie/laravel-sitemap/pull/144/commits/8450b5f9a4e09311cc2de15eddf7e23701ec386b

Reply:
And to bring a usecase: if you generate a sitemap from your menu tree and say that every level reduces the priority by `0.2` and don't want to think about if it's valid.
* Home `1.0`
  * Cat1 `0.8`
    * Cat2 `0.6`
      * Cat3 `0.4`
        * Cat4 `0.2`
          * Cat5 `0.0`
            * Articles `-0.2` clamped to `0.0`

Reply:
Or should I split it into two tests `priority_is_clamped_min` and `priority_is_clamped_max`?

Reply:
@Gummibeer this is fine!